docs/getting-started/quickstart.js-backend.mdx

```diff
@@ -397,15 +422,20 @@ The [`Auth`](/docs/reference/backend/types/auth-object) object contains importan
         const clerkClient = createClerkClient({ secretKey: process.env.CLERK_SECRET_KEY })
 
         app.get('/user', async (req, res) => {
-          // Use `getAuth()` to access `isAuthenticated` and the user's ID
+          // The `Auth` object gives you access to properties like `isAuthenticated` and `userId`
+          // Accessing the `Auth` object differs depending on the SDK you're using
+          // https://clerk.com/docs/reference/backend/types/auth-object#how-to-access-the-auth-object
```

Comment:
I like the helpfulness of adding a link here, but they are going to go unchecked if/when we do updates or redirects. 

Because of that, should we leave them or remove them?